documentation/book/ref-kafka-rack.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-rack.adoc
+
+[id='ref-kafka-rack-{context}']
+= Kafka rack awareness
+
+Kafka rack awareness can be configured in the `rack` property of `Kafka.spec.kafka`
+The `rack` object has one mandatory field named `topologyKey`.
+This key needs to match one of the labels assigned to the {ProductPlatformName} cluster nodes.
+The label is used by {ProductPlatformName} when scheduling Kafka broker pods to nodes.
+If the {ProductPlatformName} cluster is running on a cloud provider platform, that label should represent the availability zone where the node is running.
+If the {ProductPlatformName} cluster is running on a cloud provider platform, that label should represent the availability zone where the node is running.
+Usually, the nodes are labeled with `failure-domain.beta.kubernetes.io/zone` that can be easily used as `topologyKey` value.
```

Comment:
labeled > labelled
used as `topologyKey` value. >used as the `topologyKey` value.